grpc/streambuffer/streambuffer.go

```diff
@@ -93,6 +103,9 @@ func (s *Stream) recvMsg(msg interface{}) {
 
 // Run the stream
```

Comment:
This really doesn't say anything about what this function is for and how it should be used